### PR TITLE
add dashboard for external-dns

### PR DESCRIPTION
**Pull request by mcharriere**
Signed-off-by: Matias Charriere <matias@giantswarm.io>

This PR

- Adds dashboard for external-dns 

image

### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

**Comment by ubergesundheit** [comment]
Unsure if still required, but this seems to be missing the analytics https://intranet.giantswarm.io/docs/monitoring/dashboards/#adding-analytics-to-a-dashboard

File: helm/dashboards/dashboards/shared/private/external-dns.json [hunk 1]
```diff
@@ -0,0 +1,1076 @@
...
+  "graphTooltip": 0,
+  "id": 103,
+  "links": [],
+  "liveNow": false,
+  "panels": [
+    {
+      "collapsed": false,
+      "datasource": {
+        "type": "prometheus",
+        "uid": "PBFA97CFB590B2093"
```
Comment (Gacko) [inline helm/dashboards/dashboards/shared/private/external-dns.json:31]: I'm not sure, but didn't we use `"datasource": "default"` here? Maybe some @giantswarm/team-atlas knows.

**Comment by mcharriere** [comment]
> Unsure if still required, but this seems to be missing the analytics https://intranet.giantswarm.io/docs/monitoring/dashboards/#adding-analytics-to-a-dashboard

is this still valid? I get a NetworkError after adding it.

File: helm/dashboards/dashboards/shared/private/external-dns.json (hunk 1 above)
Comment (mcharriere) [reply helm/dashboards/dashboards/shared/private/external-dns.json:31]: that is the default datasource

**Review by Gacko — APPROVED** [review]

**Pull request merged**